README.md

```diff
@@ -506,19 +551,20 @@ The format is defined as a [JSON Schema](src/cs_encrypted_v1.schema.json).
+| k     | Kind              | The kind of the data (plaintext/pt, ciphertext/ct, encrypting/et).                                                                                                                                                                                                        |
+| i.t   | Table identifier  | Name of the table containing encrypted column.                                                                                                                                                                                                                            |
+| i.c   | Column identifier | Name of the encrypted column.                                                                                                                                                                                                                                             |
+| p     | Plaintext         | Plaintext value sent by database client. Required if kind is plaintext/pt or encrypting/et.                                                                                                                                                                               |
+| q     | For query         | Specifies that the plaintext should be encrypted for a specific query operation. If `null`, source encryption and encryption for all indexes will be performed. Valid values are `"match"`, `"ore"`, `"unique"`, `"ste_vec"`, `"ejson_path"`, and `"websearch_to_match"`. |
+| c     | Ciphertext        | Ciphertext value. Encrypted by proxy. Required if kind is plaintext/pt or encrypting/et.                                                                                                                                                                                  |
+| m     | Match index       | Ciphertext index value. Encrypted by proxy.                                                                                                                                                                                                                               |
+| o     | ORE index         | Ciphertext index value. Encrypted by proxy.                                                                                                                                                                                                                               |
+| u     | Unique index      | Ciphertext index value. Encrypted by proxy.                                                                                                                                                                                                                               |
+| sv    | STE vector index  | Ciphertext index value. Encrypted by proxy.                                                                                                                                                                                                                               |
```

Comment:
We reworked all the field names before I went on AL
https://github.com/cipherstash/cipherstash-suite/pull/976/files

Reply:
Ah oops. Happy to update when that change lands.